.github/workflows/flatpak-test.yml

```diff
@@ -30,6 +30,9 @@ jobs:
       - name: Build and install flatpak
         run: sh -x containers/flatpak/install --user --install-deps-from=flathub
 
+      - name: Validate metainfo
+        run: appstream-util validate org.cockpit_project.CockpitClient.metainfo.xml
```

Comment:
It's in the [appstream-util](https://packages.debian.org/search?searchon=contents&keywords=appstream-util&mode=path&suite=stable&arch=any) package, so needs to be added to apt above.

Reply:
We already do this in tools/check-dist, maybe let's just add it there?

Reply:
The problem is that we don't generate this file, except as part of the `containers/flatpak/prepare` script.  Doing it from this workflow is definitely the right place for it, and only adds one more item that we need, when we're already installing stuff anyway.

Reply:
Makes sense, thanks!